contentprovider/build.gradle

```diff
@@ -49,7 +49,7 @@ dependencies {
     })
     testCompile 'junit:junit:4.12'
 
-    compile 'org.literacyapp:literacyapp-model:1.1.56'
+    compile 'org.literacyapp:literacyapp-model:1.1.67'
```

Comment:
I think you need to change this to "compile 'ai.elimu:model:1.1.67'" as shown under "Choose dependency snippet -> Gradle" https://bintray.com/elimu-ai/maven/model

Reply:
Fixed!